swan-lake/learn/quick-tour.md

```diff
@@ -44,10 +49,12 @@ service hello on new http:Listener(9090) {
 }
 ```
 
-Now, you can run the service by running the following command.
+### Running the 'hello' Service
```

Comment:
Use the backtick instead of a single quote.  Check all the topics

Reply:
Back tick styles are not supported by the TOC plugin. Therefore, we have to use single quotes for titles.